src/javaharness/java/arcs/nativeparticles/EchoParticle.java

```diff
@@ -0,0 +1,6 @@
+package arcs.nativeparticles;
```

Comment:
We should rename directories/files with "native" in the title in a later PR. Something short but still meaningful. To bikeshed further.